_i18n/en.yml

```diff
@@ -42,47 +43,44 @@ explore:
-  answer_thirteen: <a href="https://jarbas.datasciencebr.com/" target="_blank">Jarbas</a> is a tool created by us to make it easy to visualize the data mined by our robot. It is essential for investigative work. It has emerged as an internal tool but gradually we will improve its interface aiming at a tool of public utility.
-  question_fourteen: 14. What is Rosie?
-  answer_fourteen: <a href="https://github.com/datasciencebr/rosie" target="_blank">Rosie</a> is our robot programmed to identify illegal uses of public funds, beginning with CEAP. She judges every reimbursement claimed by our congresspeople, and she tells us what are the reasons that made each one suspicions.
-  question_fifteen: 15. How was the project born?
-  answer_fifteen: The <a href="/sobre" target="_blank">first phase</a> of the project lasted two months was supported by a <a href="https://www.catarse.me/serenata" target="_blank">crowdfunding campaign</a>. Over 1296 helped us make the project hapen.
-  question_sixteen: 16. How can I contribute to the project?
-  answer_sixteen: You can contribute to the project with code, helping to maintain the team, with ideas,  <a href="https://twitter.com/search?q=%23serenatadeamor" target="_blank">retweets</a> and shares. To understand how to get started with our code base, <a href="https://datasciencebr.com/how-does-one-contribute-to-serenata-de-amor-operation-36e3e7b38207#.een62m7or" target="_blank">check this post</a>. To  directly contribute to the maintenance of the project you can tip us at <a href="https://apoia.se/serenata" target="_blank">apoia.se</a>.
-  question_seventeen: 17. What does the name Serenata de Amor mean?
-  answer_seventeen: 'There three reasons: <br>1 - We are  directly inspired by the famous Toblerone Affair – a famous case in which a politician in Sweden resigned after being caught with a simple Toblerone in the invoice of her corporate card. We want to do this: find corruption at small scale, but in large volume. <br>2 - It seems to be the a nonsense operation name typical of the Brazilian Federal Police, and this is very cool. <br>3 - It literally means love serenade — so this is our love serenade to Brazil.'
-  question_eighteen: 18. Can I use the source code of Serenata de Amor in my city, state or country?
-  answer_eighteen: Sure, this idea is awesome — and also this idea is one of the main purposes of our adoption of open source.
+  question_one: 1. What does the name Serenata de Amor mean?
+  answer_one: 'There three reasons: <br>1 - We are  directly inspired by the famous Toblerone Affair – a famous case in which a politician in Sweden resigned after being caught with a simple Toblerone in the invoice of her corporate card. We want to do this: find corruption at small scale, but in large volume. <br>2 - It seems to be the a nonsense operation name typical of the Brazilian Federal Police, and this is very cool. <br>3 - It literally means love serenade — so this is our love serenade to Brazil.'
```

Comment:
There three reasons: 

1 - We were directly inspired by the Toblerone Affair – a famous case in which a politician in Sweden resigned after being caught with a simple Toblerone she had paid with the taxpayer money. That is what we want: to monitor public expenses as small as a chocolate bar.

2 - It sounds like a nonsense operation name typical in the Brazilian Federal Police, and this is funny.
 
3 - Literally it means “love serenade” — so this is our love serenade to Brazil.